### ISSUE TITLE
using i18n.js with teaspoon ?

### ISSUE THREAD

**Issue by jvenezia** [modeset/teaspoon#298]
I'm using teaspoon with qunit to test my ember.js application, which uses [i18n.js](https://github.com/fnando/i18n-js) for internationalization.

When I run tests I get : 
```
   Uncaught ReferenceError: require is not defined
````

It is called in [this file](https://github.com/fnando/i18n-js/blob/master/app/assets/javascripts/i18n/filtered.js.erb), line 6.

Any clue on this?
Thanks!

**Comment by jejacks0n** [comment]
Have you loaded require.js, and read the documentation for the libraries you're using?

---
Jeremy Jackson

**Comment by jvenezia** [comment]
Yes, I know what I'm doing, and I searched for similar problems before submitting an issue.

I don't need require.js since i'm using rails witch uses sprockets to require files.

I've just realized that Qunit's **module** may be the problem. So [this condition](https://github.com/fnando/i18n-js/blob/master/app/assets/javascripts/i18n/filtered.js.erb#L4) is true, but it should not.

So It may be a I18n.js issue, and not teaspoon...
I'm closing this issue and checking this out...
